client/extensions/woocommerce/app/settings/payments/payment-method-stripe.js

```diff
@@ -122,38 +122,49 @@ class PaymentMethodStripe extends Component {
 		return translate( 'e.g. %(domain)s', { args: { domain } } );
 	}
 
+	areKeysAvailable = ( isLiveMode ) => {
+		const { settings } = this.props.method;
+		if ( isLiveMode ) {
+			return settings.secret_key.value.trim() && settings.publishable_key.value.trim();
```

Comment:
If secret_key, publishable_key, test_secret_key, or test_publishable_key are not present in settings, or lack string value attributes, this will bork. Could we enforce these in propTypes please?